docs/sources/helm-charts/mimir-distributed/release-notes/v4.3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Grafana Mimir Helm chart version 4.3 release notes"
+menuTitle: "V4.3 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 4.3"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 4.3 release notes
+
+Grafana Labs is excited to announce version 4.3 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.7 or Grafana Enterprise Metrics (GEM) v2.7. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.7](/docs/mimir/v2.4.x/release-notes/v2.7/) or [Grafana Enterprise Metrics v2.7](/docs/enterprise-metrics/latest/release-notes/v2-7/).
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+This is a re-release of Grafana Mimir Helm chart version 4.2 to include GEM version v2.7.
+
+It also includes the following notable change:
+
+- The ruler deployment max surge was changed from `0` to `50%`, and max unavailable from `1` to `0`.
+
+## Features and enhancements
+
+Notable enhancements are as follows:
+
+- **Cache support for GEM's admin bucket**
```

Comment:
Let's move this last in the list, most people will be more interested in the OSS features.

Reply:
Done.